sigs/fedora.md

```diff
@@ -0,0 +1,12 @@
+# Konflux + Fedora Special Interest Group
+
+## Purpose:
+
+* Operate and manage the [Fedora Konflux instance](https://gist.github.com/ralphbean/a3644111a549e8cedb0b207f90d42dc9).
+* Support people who are trying out experiments there.
+* Develop new ideas and PoCs for how Konflux might be used in Fedora.
+
+## Communication
+
+* Use the [#containers-sig](https://discussion.fedoraproject.org/tag/containers-sig) tag in Fedora Discussion for [async topics](https://discussion.fedoraproject.org/t/containers-sig-for-konflux-discussions/128061/12).
```

Comment:
Are there instructions for people not familiar with Fedora SIGs to join/follow discussions?

Reply:
That I don't know. :/

Reply:
Nothing I know of, but discussion is open, you can make an account ( accounts.fedoraproject.org ) login to discussion and set your account to follow things tagged #containers-sig